docs/usage.md

```diff
@@ -25,6 +25,7 @@ Testcontainers will try to connect to a Docker daemon using the following strate
 
 * [Temporary database containers](usage/database_containers.md) - specialized Microsoft SQL Server, MariaDB, MySQL, PostgreSQL, Oracle XE and Virtuoso container support
 * [Webdriver containers](usage/webdriver_containers.md) - run a Dockerized Chrome or Firefox browser ready for Selenium/Webdriver operations - complete with automatic video recording
+* [Kafka containers](usage/kafka_containers.md) - run a Dockerized Kafka, a distributed streaming platform
```

Comment:
Using the word `dockerized` as a verb is kind of a [generic trademark](https://en.wikipedia.org/wiki/Generic_trademark), so I think we should write it with a lowercase d. Can you please fix it in the line above as well? :slightly_smiling_face: